nil-docs/zkllvm/getting-started/writing-a-simple-circuit.mdx

```diff
@@ -0,0 +1,88 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Writing a simple circuit
+
+This tutorial describes how to create a simple circuit and use elliptic curve fields.
+
+:::tip
+
+The C++ examples in this tutorial and across the rest of the documentation use types and algorithms from [**the `crypto3` library**](https://github.com/NilFoundation/crypto3).
+
+The Rust examples use types from the forks of [**the `arkwors-rs algebra` project**](https://github.com/arkworks-rs/algebra).
+
+- [**`arkworks-algebra`**](https://github.com/NilFoundation/arkworks-algebra.git)
+- [**`arkworks-curves`**](https://github.com/NilFoundation/arkworks-curves.git)
+
+:::
+
+## Multiplication example
+
+Any circuit is a series of polynomials which, in the case of high-level programming languages, can be expressed as a function.
+
+For zkLLVM to correctly recognise a function as a circuit, it needs to be preceeded by the following directive.
+
+<Tabs groupId='language'>
+  <TabItem value='cpp' label='C++'>
+
+    ```cpp
+    [[circuit]]
+    ```
+
+  </TabItem>
+
+  <TabItem value='rust' label='Rust'>
+
+    ```rust
+    #[circuit]
+    ```
+
+  </TabItem>
+</Tabs>
+
+A simple example of a circuit with one multiplication operator would look as follows.
+
+<Tabs groupId='language'>
+  <TabItem value='cpp' label='C++'>
+
+    ```cpp
+
+    [[circuit]] int multiplication_example(
+    int a,
+    int b) {
+      auto c = a*b;
+      return c;
+    }
+
+    ```
+
+  </TabItem>
+
+  <TabItem value='rust' label='Rust'>
+
+    ```rust
+    #[circuit]
+    pub fn multiplication_example(a: i32, b: i32) -> i32 {
+      let c = a * b;
+      c
+    }
+    ```
+
+  </TabItem>
+</Tabs>
+
+:::info[Using standard types in C++]
+
+The compiler supports several types from the `std` library such as `int` and `long`.
```

Comment:
I suggest to remove this sentence at all and leave only the second one in the note. `int` and `long` types does not belong to `std`.

Reply:
you are correct, sentence is removed